app/models/form.rb

```diff
@@ -111,8 +111,8 @@ def next_incomplete_section_redirect_path(subsection, log)
     when :in_progress
       "#{next_subsection.id}/check_answers".dasherize
     when :not_started
-      first_question_in_subsection = next_subsection.pages.find { |page| page.routed_to?(log, nil) }.id
-      first_question_in_subsection.to_s.dasherize
+      first_question_in_subsection = next_subsection.pages.find { |page| page.routed_to?(log, nil) }
+      first_question_in_subsection ? first_question_in_subsection.id.to_s.dasherize : next_incomplete_section_redirect_path(next_subsection, log)
```

Comment:
the is the site of the original error found.
It is possible that in the next subsection there are not questions routed to, in this case we should link the next subsection that does have questions routed to.

Reply:
Hmm, I wonder if it would make sense so that `next_subsection` returns the next subsection that is routed to, but I'm not sure where else this method is used, so happy for you to go with what makes more sense